test/helpers/utils.go

```diff
@@ -635,3 +635,19 @@ func SkipK8sVersions(k8sVersions string) bool {
 	constraint := versioncheck.MustCompile(k8sVersions)
 	return constraint(k8sVersion)
 }
+
+// DualStackSupported returns whether the current environment has DualStack IPv6
+// enabled or not for the cluster.
+func DualStackSupported() bool {
+	k8sVersion := GetCurrentK8SEnv()
+	switch k8sVersion {
+	// Add kubernetes versions for which we add dual stack support.
+	case "1.18", "1.19", "1.20":
```

Comment:
Please invert the logic, i.e., set `case "1.12","1.13","1.14","1.15","1.16","1.17":` so that we don't need to add a new version each time a k8s verion is released.

Reply:
Yeah, makes sense.